### PR TITLE
Fix batchSize for deleting datapoints

### PR DESCRIPTION
**Pull request by exyi**
While debugging other issue in Jetfire, I run into this limitation.

### CODE REVIEW

File: src/main/scala/cognite/spark/v1/DefaultSource.scala [hunk 1]
```diff
@@ -188,8 +188,12 @@ class DefaultSource
...
       }
       val batchSizeDefault = relation match {
-        case _: NumericDataPointsRelationV1 => Constants.CreateDataPointsLimit
-        case _: StringDataPointsRelationV1 => Constants.CreateDataPointsLimit
+        case _: NumericDataPointsRelationV1 | _: StringDataPointsRelationV1 =>
+          if (config.onConflict == OnConflict.Delete) {
+            Constants.DefaultDataPointsLimit
+          } else {
+            Constants.CreateDataPointsLimit
+          }
```
Comment (emelvarol-cognite) [inline src/main/scala/cognite/spark/v1/DefaultSource.scala:196]: why

Comment (exyi) [reply src/main/scala/cognite/spark/v1/DefaultSource.scala:196]: The datapoint endpoint allows 100_000 items when creating, but only 10_000 when deleting

Comment (emelvarol-cognite) [reply src/main/scala/cognite/spark/v1/DefaultSource.scala:196]: can you add a comment on this?

Comment (emelvarol-cognite) [reply src/main/scala/cognite/spark/v1/DefaultSource.scala:196]: in case someone wonders why in the future

**Review by emelvarol-cognite — APPROVED** [review]

**Pull request merged**